common/djangoapps/course_modes/models.py

```diff
@@ -692,3 +697,38 @@ class CourseModesArchive(models.Model):
     expiration_date = models.DateField(default=None, null=True, blank=True)
 
     expiration_datetime = models.DateTimeField(default=None, null=True, blank=True)
```

Comment:
Sure, but in that case the signal code would set `_expiration_datetime` (the private version). I'm just thinking about future-proofing this code from developers who don't know the history that they're working with.